src/bepasty/static/app/js/utils.js

```diff
@@ -8,6 +8,35 @@ $(function () {
         });
     });
 
+    // Show a edit dialog box when trying to edit metadata
+    $("#edit-btn").click(function() {
+        var form_name = "edit-frm"
+        var hidden_name_id = "#hidden-" + form_name
+        // Read form template from html
+        var modal_form = $(hidden_name_id).html();
+        var modal_title = $(hidden_name_id).attr('modalTitle');
+        var modal_focus = $(hidden_name_id).attr('modalFocus');
+        // A bit hack, to avoid implement with .dailog() from scratch
+        var box = bootbox.confirm({
+            title: modal_title,
+            message: modal_form,
+            centerVertical: true,
+            onShown: function(e) {
+                if (modal_focus) {
+                    $(this).find("#" + modal_focus).trigger('focus');
+                }
+                // Support jquery-ui autocomplete
+                contenttype_autocomplete(this)
+            },
+            callback: function(result) {
+                // Note, if pushed Enter key on input boxes, this is not called
```

Comment:
please note that this is not called when hitting the Enter key on the input box.

Reply:
fixed